src/client/Client.cc

```diff
@@ -10154,12 +10540,13 @@ int Client::pwritev(int fd, const struct iovec *iov, int iovcnt, int64_t offset)
 {
   if (iovcnt < 0)
     return -CEPHFS_EINVAL;
-  return _preadv_pwritev(fd, iov, iovcnt, offset, true);
+  return _preadv_pwritev(fd, iov, iovcnt, offset, true, nullptr);
 }
 
 int64_t Client::_preadv_pwritev_locked(Fh *fh, const struct iovec *iov,
                                        unsigned iovcnt, int64_t offset,
-                                       bool write, bool clamp_to_int)
+                                       bool write, bool clamp_to_int,
+                                       CC_Onfinish *onfinish)
```

Comment:
You can just set `CC_Onfinish *onfinish=nullptr` and then there is no need to change the irrelevant code.